### ISSUE TITLE
Preserve scroll offset in Tabs page when restarting Fenix

### ISSUE THREAD

**Issue by Cheap-Skate** [mozilla-mobile/fenix#5804]
If I scroll the Tabs page to get rid of the Firefox Preview logo and just see my tabs.

-> The scroll offset is preserved if I move into a tab and then back to the Tabs page

-> The scroll offset is NOT preserved if I quit & restart Fenix.

Could the scroll offset be preserved in the latter case? More consistent, better muscle memory, don't need to look at the Preview logo, blah blah.

**Comment by mikeGolf08** [comment]
Hi Cheap-Skate,

I spent a few moments to look after your requested feature and from my side I think it does not make sense implementing it - Let me explain why:

The tab offset on Fenix's homescreen is kept when simply "minimising the app" but not when tapping *Quit* because *Quit* is only visible if you *want to delete browsing data on quit* and if data is beiing deleted it is like a reset and Fenix shows the initial screen next time.

In case you just delete cookies for example you are right because one expect no change on tab offset but I think this is not the intension of "delete browsing data on quit" and is *not necessary to implement*.

I don't know how the code looks like for that method but it was changed some time ago.

If you do not want to see the Fenix logo on homescreen that is an *UX improvement* ticket - I am sure somebody of the UX team will look into it in no time!
